doc/source/whatsnew/v0.21.0.txt

```diff
@@ -25,6 +25,10 @@ New features
 - Added `__fspath__` method to :class`:pandas.HDFStore`, :class:`pandas.ExcelFile`,
   and :class:`pandas.ExcelWriter` to work properly with the file system path protocol (:issue:`13823`)
 
+- ``SparseSeries`` and ``SparseArray`` now support 1d ``scipy.sparse.spmatrix`` in constructor.
+  Additionally, ``SparseDataFrame`` can be assigned columns of ``scipy.sparse.spmatrix``;
```

Comment:
make this 2nd sentence a separate bullet point (you can use same issue on both of them, or 2nd one should be the PR number maybe)